test/shopify_app/configuration_test.rb

```diff
@@ -2,6 +2,18 @@
 
 require "test_helper"
 
+module Shopify
+  class CustomPostAuthenticateTasks
```

Comment:
This is just a question for my own benefit: will any class that defines a `perform` method do here? Should we check if it extends a job class instead to standardize things a bit more?

Reply:
I also don't think it matters too much, but I remember the documentation said it could "run this later", which I took to mean it would deal with a `perform_later` type of thing. But just for the sake of standardization, I don't think it's necessary.